### ISSUE TITLE
MODX 3 No access policies or policy templates

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#14696]
### Summary
Access policies and policy templates disappeared in the manager panel, see screen.

![policies](https://user-images.githubusercontent.com/12523676/63636609-a2bf3980-c682-11e9-856a-01429978bd52.png)

And errors appear in the console:

```
[2019-08-24 14:15:41] (ERROR in xPDO\xPDO::call @ /home/a/aleeviz8/m3/public_html/core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 759) MODX\Revolution\modX::loadClass is deprecated since version 3.0. Replace references to class modAccessPolicyTemplate with MODX\Revolution\modAccessPolicyTemplate to take advantage of PSR-4 autoloading.
[2019-08-24 14:15:41] (ERROR in xPDO\xPDO::getFields @ /home/a/aleeviz8/m3/public_html/core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 1401) MODX\Revolution\modX::loadClass is deprecated since version 3.0. Replace references to class modAccessPolicyTemplate with MODX\Revolution\modAccessPolicyTemplate to take advantage of PSR-4 autoloading.
[2019-08-24 14:15:41] (ERROR @ /home/a/aleeviz8/m3/public_html/core/vendor/xpdo/xpdo/src/xPDO/Om/xPDOObject.php : 227) Error 42S22 executing statement: 
Array
(
    [0] => 42S22
    [1] => 1054
    [2] => Unknown column 'MODX\Revolution\modAccessPolicyTemplate.name' in 'order clause'
)
```

### Environment
MODX 3

**Comment by Mark-H** [comment]
Prepeared PR #14706 to fix this - would be great if you could confirm.

**Comment by Ibochkarev** [comment]
Fixed in #14706 @modxbot close
